node/rate_limiter.js

```diff
@@ -72,7 +75,18 @@ function RateLimiter(options) {
     }
     var self = this;
 
+    // stats
+    self.statEmitter = options.statEmitter || new StatEmitter();
+    self.rateLimiterEmptyTags = new stat.RateLimiterEmptyTags();
+    self.serviceRpsStat = self.statEmitter.defineGauge('tchannel.rate-limiting.service-rps');
```

Comment:
gauges are expensive because we dont aggregate them due to a stat bug. 

but 4 per second should be ok.